mrp_routing_account_journal/view/view.xml

```diff
@@ -11,18 +11,5 @@
                 </xpath>
             </field>
         </record>
-        <record id="mrp_bom_journal_form_view" model="ir.ui.view">
-            <field name="name">mrp.bom.journal.form.view</field>
-            <field name="inherit_id" ref="mrp.mrp_bom_form_view"/>
-            <field name="model">mrp.bom</field>
-            <field name="arch" type="xml">
-                <xpath expr="//label[@for='routing_id']" position="before">
-                    <label for="journal_id" class="oe_inline" groups="account.group_account_manager"/>
-                    <div groups="account.group_account_manager">
```

Comment:
@nhomar here lies the problem of your comments on ACL.

there were no problems of ACL but problems for ACL on Views not properly addressed